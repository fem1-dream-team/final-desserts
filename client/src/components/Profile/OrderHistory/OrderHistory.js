import React, {Fragment, useEffect, useState} from 'react';
import classes from './OrderHistory.module.css';
import {connect} from 'react-redux';
import OrderHistoryItem from "./OrderHistoryItem/OrderHistoryItem";
import {Link} from "react-router-dom";
import orderReducer from "../../../reducers/orderReducer";

const OrderHistory = (props) => {
	orderReducer();
	console.log(props);
	/*const productItems = props.orders;
		const productList = productItems.map(item => {
		const id = item.ordNo;
		return(
			<OrderHistoryItem
	ordNo={item.ordNo}
	creationDate={item.creationDate}
	deliveryStatus={item.deliveryStatus}
	key={id}
	state={props}
/>
		)
	});*/

	return (
		<div className={classes.orderWrap}>
			<h3>Order history</h3>
			<div>
				{/*{productList}*/}

			</div>
		</div>
	)
}
const mapStateToProps = (state) => {
	return ({
		orders: state.orders
	})
}

export default connect(mapStateToProps, {})(OrderHistory)




/*========================*/
/*<OrderHistoryItem
	ordNo={item.ordNo}
	creationDate={item.creationDate}
	deliveryStatus={item.deliveryStatus}
	key={id}
	state={props}
/>*/

/*
<div
ordNo={item.ordNo}
creationDate={item.creationDate}
deliveryStatus={item.deliveryStatus}
key={id}
state={props}
/>*/
