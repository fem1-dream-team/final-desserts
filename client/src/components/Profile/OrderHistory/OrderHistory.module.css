.orderWrap {
    grid-area: profile;
    padding: 25px;
}

.title{
    font-size: 1.3em;
    padding: 5px;
}