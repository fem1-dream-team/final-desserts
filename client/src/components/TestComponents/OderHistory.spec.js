import React from "react";
import OrderHistory from "../Profile/OrderHistory/OrderHistory";
import renderer from 'react-test-renderer'

describe("OrderHistory component", () => {

	it("OrderHistory component render", () => {
		const wrapper = renderer.create(<OrderHistory />).toJSON();
		expect(wrapper).toMatchSnapshot();

	});
});