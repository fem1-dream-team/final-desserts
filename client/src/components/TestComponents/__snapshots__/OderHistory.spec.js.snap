// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderHistory component OrderHistory component render 1`] = `
<div
  className="item"
>
  <h3>
    here will be the order history
  </h3>
</div>
`;
